### ISSUE TITLE
Still maintained?

### ISSUE THREAD

**Issue by IzzySoft** [murgo/IrssiNotifier#206]
Just wondering whether this project is still maintained, @murgo – with no commits for a year and issues staying unanswered. Not complaining, just checking the status. Could you please clarify? Thanks!

**Comment by murgo** [comment]
Yeah, still maintained but since it's a hobby project used by many I rarely
do more than necessary fixes for the app.

Granted, there are some stuff that could be merged or implemented, but the
current version works fine for majority of users so I haven't priorized
changes very high.

la 2. marrask. 2019 klo 16.53 Izzy <notifications@github.com> kirjoitti:

**Comment by IzzySoft** [comment]
Thanks for the prompt reply! I just checked the top-most ones for "last answers" from "the team" – not saying it were plenty of unanswered issues, just that I didn't see "obvious activity". Good to know then it's not abandoned.

Maybe it would be a good idea to point that out in the Readme (like, mature status, still maintained, rare releases)?

**Comment by volans-** [comment]
@murgo Thanks a lot for this app, I've been using it since long time!
Now that I've updated my phone to Android 14, even if it seems to work, every time I open the app I get a popup that says:

> This app was built for an older version of Android. It might not work properly and doesn't include the latest security and privacy protections. Check for an update or contact the app's developer.

Do you have any plan to make a release to fix this?
Given the app seems to still work maybe there aren't too many changes to make Android happy.

**Comment by murgo** [comment]
Yeah, it's too bad it's kind of in need of upgrade but I haven't had time to do it. 

I spent a weekend looking into raising the API levels to latest to make Google Play happy, but it wasn't very straightforward, bunch of stuff is stale / obsolete / deprecated (even though the app is quite simple). I can't recall specifics since it was several months ago.

I do intend to upgrade the app so it plays nicely with Play store again, hopefully this year :P. PRs are welcome if you want to give upgrading the app a shot (who knows, maybe it's not as bad as I thought).

I still use IRC and IrssiNotifier though. Thanks for the interest and the message, keep on ircing.

**Comment by AVaccNY** [comment]
Here's a vote for the upgrade. :) I didn't think to ask. I figured I would finally not install it on my shiny new S24.. It would be great to know when there is a storm in #PiratesIRC tho ;))) 

Seriously, though this comment is just to let you know there are more of us than you may have thought. Thank you for all you have done.
